core/src/main/scala/com/evolutiongaming/catshelper/EffectHelper.scala

```diff
@@ -107,16 +107,16 @@ object EffectHelper {
 
   implicit class FOpsEffectHelper[F[_], A](val self: F[A]) extends AnyVal {
 
-    def redeem[B, E](recover: E => B, map: A => B)(implicit bracket: Bracket[F, E]): F[B] = {
+    def redeem[B, E](recover: E => B, map: A => B)(implicit bracket: MonadCancel[F, E]): F[B] = {
       bracket.redeem(self)(recover, map)
     }
 
-    def redeemWith[B, E](recover: E => F[B], flatMap: A => F[B])(implicit bracket: Bracket[F, E]): F[B] = {
+    def redeemWith[B, E](recover: E => F[B], flatMap: A => F[B])(implicit bracket: MonadCancel[F, E]): F[B] = {
```

Comment:
could you please rename everywhere

```implicit bracket: MonadCancel```
to 

```implicit monadCancel: MonadCancel```

Reply:
done